homeassistant/components/opentherm_gw/sensor.py

```diff
@@ -1,20 +1,621 @@
+        ),
+    ),
+    (
+        [gw_vars.BOILER, gw_vars.THERMOSTAT],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.DATA_SLAVE_PRODUCT_TYPE,
+            friendly_name_format="Boiler Product Type {}",
+        ),
+    ),
+    (
+        [gw_vars.BOILER, gw_vars.THERMOSTAT],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.DATA_SLAVE_PRODUCT_VERSION,
+            friendly_name_format="Boiler Product Version {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_MODE,
+            friendly_name_format="Gateway/Monitor Mode {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_DHW_OVRD,
+            friendly_name_format="Gateway Hot Water Override Mode {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_ABOUT,
+            friendly_name_format="Gateway Firmware Version {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_BUILD,
+            friendly_name_format="Gateway Firmware Build {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_CLOCKMHZ,
+            friendly_name_format="Gateway Clock Speed {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_LED_A,
+            friendly_name_format="Gateway LED A Mode {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_LED_B,
+            friendly_name_format="Gateway LED B Mode {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_LED_C,
+            friendly_name_format="Gateway LED C Mode {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_LED_D,
+            friendly_name_format="Gateway LED D Mode {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_LED_E,
+            friendly_name_format="Gateway LED E Mode {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_LED_F,
+            friendly_name_format="Gateway LED F Mode {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_GPIO_A,
+            friendly_name_format="Gateway GPIO A Mode {}",
+        ),
+    ),
+    (
+        [gw_vars.OTGW],
+        OpenThermSensorEntityDescription(
+            key=gw_vars.OTGW_GPIO_B,
+            friendly_name_format="Gateway GPIO B Mode {}",
+        ),
+    ),
```

Comment:
What does this return?

Reply:
Hot Water Override Mode: `0`/`1` (override active in off/on mode) or `A` (override inactive)

Firmware version/build/clock speed: Info about the OpenTherm Gateway firmware/hardware. Could possibly be integrated in device info?

LED/GPIO modes: configured modes for the different LEDs and GPIO pins on the gateway. LED mode is one of `RXTBOFHWCEMP` and GPIO mode is in the range `0` through `8`.

Reply:
Sounds like good examples for enum sensors as well